docs/03-extensions/theme-chocolatine/reference/css-variables.mdx

```diff
@@ -97,3 +97,18 @@ variables are:
 - `--boxSizeMargin`
 - `--smallContainerWidth`
 - `--containerWidth`
+
+## Check for unused variables and custom properties
+
+Front-Commerce provides a script to check for unusued SCSS variables and custom
```

Comment:
Shouldn't it be only about exported ones?
```suggestion
Front-Commerce provides a script to check for unusued exported CSS variables and custom
```

Reply:
It checks for SCSS _and_ CSS properties, both exported or not (as discussed on slack).

Reply:
Ok, thanks.